dao/src/test/java/org/thingsboard/server/dao/sql/user/JpaUserDaoTest.java

```diff
@@ -45,35 +46,54 @@
  */
 public class JpaUserDaoTest extends AbstractJpaDaoTest {
 
+    // add to count user sysadmin@thingsboard.org
+    final int COUNT_CREATED_USER = 1;
+    final int COUNT_SYSADMIN_USER = 90;
+    UUID tenantId;
+    UUID customerId;
     @Autowired
     private UserDao userDao;
 
+    @Before
+    public void setUp() {
+        tenantId = Uuids.timeBased();
+        customerId = Uuids.timeBased();
+        create30TenantAdminsAnd60CustomerUsers(tenantId, customerId);
+    }
+
+    @After
+    public void tearDown() {
+        delete30TenantAdminsAnd60CustomerUsers(tenantId, customerId);
+    }
+
     @Test
-    @DatabaseSetup("classpath:dbunit/user.xml")
     public void testFindAll() {
         List<User> users = userDao.find(AbstractServiceTest.SYSTEM_TENANT_ID);
-        assertEquals(users.size(), 5);
+        assertEquals(users.size(), COUNT_CREATED_USER + COUNT_SYSADMIN_USER);
     }
 
     @Test
-    @DatabaseSetup("classpath:dbunit/user.xml")
-    public void testFindByEmail() {
-        User user = userDao.findByEmail(AbstractServiceTest.SYSTEM_TENANT_ID,"sysadm@thingsboard.org");
-        assertNotNull("User is expected to be not null", user);
-        assertEquals("9cb58ba0-27c1-11e7-93ae-92361f002671", user.getId().toString());
-        assertEquals("c97ea14e-27c1-11e7-93ae-92361f002671", user.getTenantId().toString());
-        assertEquals("cdf9c79e-27c1-11e7-93ae-92361f002671", user.getCustomerId().toString());
-        assertEquals(Authority.SYS_ADMIN, user.getAuthority());
-        assertEquals("John", user.getFirstName());
-        assertEquals("Doe", user.getLastName());
+    public void testFindByEmail() throws JsonProcessingException {
+        User user = new User();
+        user.setId(new UserId(UUID.fromString("cd481534-27cc-11e7-93ae-92361f002671")));
```

Comment:
there is a much cleaner method exist. please, use it 
`UserId fromString(String userId)`